posthog/temporal/batch_exports/utils.py

```diff
@@ -24,3 +29,33 @@ def rewind_gen() -> collections.abc.Generator[T, None, None]:
             yield i
 
     return (first, rewind_gen())
+
+
+async def try_set_batch_export_run_to_running(run_id: str | None, logger, timeout: float = 10.0) -> None:
```

Comment:
This default timeout is kind of arbitrary, but updating this row when starting and finishing usually takes a handful of ms only, so 10 secs should be more than plenty, if not too much.